packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -157,6 +159,32 @@ export default class TelemetryReporter extends vscode.Disposable {
     }
   }
 
+  public sendExceptionEvent(
+    exceptionName: string,
+    exceptionMessage: string,
+    measurements?: { [key: string]: number }
```

Comment:
Do we also want the option to send properties in addition to measurements?

Reply:
Eventually yes, that's available in the latest version of application insights but I wanted to keep the changes as small as possible for now. I also didn't see a current need for sending properties so I decided to leave updating the application insights version in a future PR.